src/cloud-api-adaptor/versions.yaml

```diff
@@ -21,7 +21,7 @@ cloudimg:
 tools:
   bats: 1.10.0
   golang: 1.21.10
-  rust: 1.72.0
+  rust: 1.74.0
```

Comment:
This is a slighty tricky one as we now have rust 1.74 for building the KBS client and 1.72 for building the kata-agent. I don't really like the discrepancy between the version that we use and upstream kata does, but as long as it works I'm ok with it as we plan to use the kata-agent cache soon which should remove that.